root/statistics/utilities.js

```diff
@@ -35,10 +35,11 @@ export const TimelineLink = ({
   statName,
 }: {statName: string}): React.Element<'a'> => (
   <a
-    href={'/statistics/timeline/' + statName}
+    href={'/statistics/timeline/' + encodeURIComponent(statName)}
     title={l('See on timeline')}
   >
     <img
+      alt=""
```

Comment:
I guess it’s set empty because it is enclosed in a `a` tag with non-empty `title` attribute.